trophic_levels-biomass_indicators.rmd

```diff
@@ -210,11 +210,11 @@ totalcarnivores<-carnivorerast[[49:60]]
 
```

Comment:
Here the years are not visible in the headings of the 12 panels. Would be great to see them.